src/install/dracut-install.c

```diff
@@ -1486,6 +1503,7 @@ static void usage(int status)
                "  -d --dir          SOURCE is a directory\n"
                "  -l --ldd          Also install shebang executables and libraries\n"
                "  -L --logdir <DIR> Log files, which were installed from the host to <DIR>\n"
+               "     --pretend      Don't actually copy files, just show what would be installed\n"
```

Comment:
This is usually called `--dry-run` by tools (with `-n` as a shortcut).

Reply:
I think I prefer `--dry-run` over `--list` because it's following all the same code paths but just not doing the actual copy.